build.csx

```diff
@@ -0,0 +1,225 @@
+#load "packages/simple-targets-csx.5.2.0/simple-targets.csx"
+
+#r "System.Xml.Linq"
+
+using System.Runtime.Serialization.Json;
+using System.Text.RegularExpressions;
+using System.Xml;
+using System.Xml.Linq;
+using static SimpleTargets;
+
+static var solutionName = "FakeItEasy";
+
+static var solution = "./" + solutionName + ".sln";
+static var packagesDir = Path.GetFullPath("packages");
+var repoUrl = "https://github.com/FakeItEasy/FakeItEasy";
+var assemblyInfo = "./src/CommonAssemblyInfo.cs";

```

Comment:
I only added the static when needed. I was on the fence. It makes no difference to me whether they're there or not, so I'll add 'em.